docs/FAQ.md

```diff
@@ -109,10 +109,41 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I deal with divergent changes ('??' after the ID)?
+
+A [divergent change][glossary_divergent_change] represents a change that has two
+or more visible commits associated with it. To refer to such commits, you must
+use their [commit ID][glossary_commit_id]. Most commonly, the way to resolve
+this is to abandon the unneeded commits (using `jj abandon <commit ID>`). If you
+would like to keep both revisions of the change, you can `jj duplicate` one of
```

Comment:
(Optional) Instead of "both revisions of the change", I'd say "both versions of the commit". As I said before, I find the word "change" a bit ambiguous on its own.

Reply:
Thanks. I think I'll use "both commits with this change id". "Both versions of the commit" sounds more confusing to me (in my mind, they are two separate commits, not versions of the same commit 😅)